modules/arhParser.jsm.js

```diff
@@ -184,6 +193,13 @@ function parseResInfo(str) {
 		res.reason = reg_match[1] || reg_match[2];
 	}
 
+	// Outlook specific action (optional)
+	// https://learn.microsoft.com/en-us/defender-office-365/message-headers-eop-mdo
+	if (prefs.getBoolPref("relaxedParsing")) {
+		const actionspec_p = `action${rfcParser.get("CFWS_op")}=${rfcParser.get("CFWS_op")}${rfcParser.get("value_cp")}`;
+		reg_match = match_o(str, actionspec_p);
+	}
```

Comment:
I'm not sure, why this code was added, as it seems to have no effect on the result?

Reply:
Try the different headers from the following test case, which failed for me without this

https://github.com/lieser/dkim_verifier/blob/72b0716bd21f53d629a9afbfdf8795596e133310/test/unittest/arhParserSpec.mjs.js#L249-L276

Reply:
Ah, okay. Thank you for the explanation.